### PR TITLE
Major update

### PR DESCRIPTION
**Pull request by rickerd**
- PHP5 standards in class Database.php
- Added properties to top of class with Docblock
- Added phpunit.xml.dist to autoload vendor/autoload.php and add colors to Unittest + specify the tests folder
- Updated the __construct() to only do a single thing
- Added private method makeDir() to be able to make dirs and not use @ to suppress any messages

### CODE REVIEW

**Comment by rickerd** [comment]
PHPunit still working like normally.

---

PHPUnit 3.7.38 by Sebastian Bergmann.

Configuration read from ***/microdb/phpunit.xml.dist

............

Time: 1.04 seconds, Memory: 3.50Mb

OK (12 tests, 13 assertions)

File: src/MicroDB/Database.php [hunk 1]
```diff
@@ -2,38 +2,55 @@
...
 namespace MicroDB;
 
+use Exception;
+
 /**
  * A file-based JSON object database
  */
-class Database {
+class Database
+{
```
Comment (morris) [inline src/MicroDB/Database.php:12]: the { should be on the same line as class

File: tests/DatabaseTest.php [hunk 2]
```diff
@@ -1,230 +1,251 @@
...
-	function testValidId() {
-		$a = array();
-		$a[] = self::$db->load(null);
-		$a[] = self::$db->load('');
-		$a[] = self::$db->load(0);
-
-		$ex = array(null, null, null);
-		$this->assertEquals($ex, $a);
-	}
+class DatabaseTest extends PHPUnit_Framework_TestCase
```
Comment (morris) [inline tests/DatabaseTest.php:10]: the tests should use the same code style as the library

File: src/MicroDB/Database.php [hunk 3]
```diff
@@ -496,16 +502,26 @@ function trigger($event, $args = null) {
 
 	/**
 	 * Split event keys by whitespace and/or comma
+	 * @param $events
+	 * @return array
 	 */
-	protected function splitEvents($events) {
-		if(is_array($events))
+	private function splitEvents($events) {
```
Comment (morris) [inline src/MicroDB/Database.php]: protected should be enough

**Comment by morris** [comment]
Great work, there's a few issues I pointed out. Would be cool if you fix these

File: src/MicroDB/Database.php (hunk 1 above)
Comment (rickerd) [inline src/MicroDB/Database.php:12]: Sorry, but according to the PSR-1 standards the { should be on the next line.
http://www.php-fig.org/psr/psr-1/

**Comment by rickerd** [comment]
I've updated the things to withstand your needs.
Just trying to get the code to a higher level and develop myself

**Comment by morris** [comment]
Uhm, are we missing a commit? It still seems that the test suite would be PSR-2 but everything else is my own code style. I'm open to changing everything to PSR-2, but having multiple code styles in such a small library is a bad idea. How is PSR-2 adoption out there? Let's decide on a certain code style and then apply it everywhere :-)

**Comment by rickerd** [comment]
True, i can imagine that. I've put all the PSR-2 standards into the files

**Comment by morris** [comment]
Alright, let's go with PSR-2 then. Looking forward to more contributions :)

**Comment by rickerd** [comment]
More contributions by me or just in general? haha
Ive seen some things that are strange or unknown to me.
So maby we can take these things also to a higher level.

Never theless i really love your MicroDB. Its quick and works like a charm.

I also like it that you are open for changes, i just had to say that to you. Appriciate it :)

**Comment by morris** [comment]
Both, i guess. If you find anything or have questions, just open an issue so we can discuss. I'm glad you like it :-)

**Pull request merged**